app/views/news/2018-09-06-august-2018-monthly-update.html.md

```diff
@@ -0,0 +1,94 @@
+
+We’re excited to enter a new phase as a “grown up” non-profit, and we’re looking forward to being able to support the Ruby open source community for years to come. 💎💝
+
+## bundler news
+
+With the help of [@eanlain](http://github.com/eanlain), we shipped a new guide: [“How to use Bundler with Docker”](https://bundler.io/v1.16/guides/bundler_docker_guide.html). We also dramatically improved error messages when version requirements conflict, shipped a playbook for adding or removing core team members, and fixed some issues handling gemspecs with non-ASCII characters. We also merged a fix that could cause Bundler to [fail when trying to install a gem that has had a version yanked recently](https://github.com/bundler/bundler/pull/6675).
+
+On top of that code work, we also added two new contributors to [the Bundler team](https://bundler.io/contributors.html)! Welcome to [Stephanie Morillo](https://www.twitter.com/radiomorillo) and [Grey Baker](https://twitter.com/greybaker).
+
+Stephanie has helped create new Bundler docs (like the troubleshooting RubyGems and Bundler TLS/SSL issues guide), and update existing ones. She’s also responsible for updating the Bundler contributor guidelines and trying to make Bundler docs more accessible to new contributors. We're excited to have her on board, and looking forward to working with her more in the future.
+
+Grey is the author of [Dependabot](https://dependabot.com), and has regularly contributed to [Molinillo](https://github.com/cocoapods/molinillo), the core resolver library that powers Bundler, RubyGems, and Cocoapods.
+
+In total this month, Bundler gained 58 new commits, contributed by 10 authors. There were 334 additions and 54 deletions across 30 files.
+
+## rubygems.org news
+
+In August, we blacklisted several gems with names that were close to other popular gems, in response to CVE-2018-3779. We're starting to investigate ways to protect RubyGems.org from malicious gems—if you're interested in helping work on that, let us know! We also reviewed and merged performance improvements to the “rubygems#show” and “version#index” pages, contributed by [@nateberkopec](https://github.com/nateberkopec).
+
+In total, RubyGems.org gained 11 commits from 5 authors, making 44 additions and 35 deletions across 8 files.
+
+## rubygems news
+
+In RubyGems, we fixed some bugs, fixed some tests, and made sure that gems with `allowed_push_host` set will be pushed to the correct host by `gem push`. In total there were 19 new commits, contributed by 5 authors, with 112 additions and 26 deletions across 13 files.
+
```

Comment:
Maybe we can add this too? [Fix auto resign expired certificate](https://github.com/rubygems/rubygems/pull/2380)

Reply:
Sounds good! Adding it in now.